README.md

```diff
@@ -20,3 +20,15 @@ docker run --name buildAgent1 -d --restart=always --storage-opt size=50G -e AGEN
 
 The agent gets its name from the environment variable `AGENT_NAME`. This name will showup in the teamcity web ui. If you don't set
 the name you will endup with a agent called `placeholder`.
+
+# JDK selection
+
+By default, the openjdk 8 will be used. Set env vars `PATH` and `JAVA_HOME` to select a jetbrains jdk:
+
+```
+# for jetbrains java 8
+export JAVA_HOME="/usr/lib/jvm/${JB_JAVA8_VERSION}/" PATH="/usr/lib/jvm/${JB_JAVA8_VERSION}/bin:$PATH"
```

Comment:
why don't we use `JB_JAVA11_HOME` and `JB_JAVA8_HOME` here? 

IMHO we should also link to https://github.com/mbeddr/mps-gradle-plugin#runantscript and mention the `executable` task property. As you typically don't want to do manipulate the path before running the gradle scripts.